### ISSUE TITLE
Error: No tag TODAY

### ISSUE THREAD

**Issue by sdruskat** [johannesjo/super-productivity#702]
> EDIT: Could be a duplicate of #701, which I didn't see because *create issue* start right up when clicking on `Copy & Report` in the log view itself (which is great functionality, but may lead to these duplicates I assume).

### Steps to Reproduce

*This is on my Windows machine:*

1. `Save & go home` on Friday night
2. Start SP via the Start menu on Monday morning
3. 💥 (Is it perhaps a Monday bug?)
4. (After closing SP after a few minutes and re-stating, the DB is back, and SP runs as expected. **I assume this may have to do with starting SP before all services are ready right after Windows starts?**)

### Error Log (Desktop only)

Note on the comment in the issue template:
```markdown
<!-- For the desktop versions, there is also an error log file in case there is no console output.
Usually, you can find it here:
on Linux:
~/.config/superProductivity/log.log
--or--
~/snap/superproductivity/current/.config/superProductivity/log.log
on macOS: ~/Library/Logs/superProductivity/log.log
on Windows: %USERPROFILE%\AppData\Roaming\superProductivity\log.log
. -->
```

There is no `log.log` in the AppData folder root for SP, there is `main.log` in the `logs/` folder though:

```log
[2020-11-30 08:29:33.453] [error] Frontend Error: Error: No tag TODAY
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2028559
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144622
    at memoized (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144116)
    at lt (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2145023)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144676
    at memoized (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144116)
    at a.project (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2143846)
    at a._next (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:1937661)
    at a.next (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:220941)
    at t._subscribe (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:111996) Error: No tag TODAY
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2028559
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144622
    at memoized (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144116)
    at lt (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2145023)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144676
    at memoized (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144116)
    at a.project (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2143846)
    at a._next (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:1937661)
    at a.next (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:220941)
    at t._subscribe (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:111996)
[2020-11-30 08:29:41.288] [error] Frontend Error Stack: webpack:///src/app/features/tag/store/tag.reducer.ts:46:12
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1096:29)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
args (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1059:38)
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1105:35)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
source$.pipe(pluck (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:868:24)
_next (webpack:///node_modules/rxjs/_esm2015/internal/operators/map.js:29:34)
next (webpack:///node_modules/rxjs/_esm2015/internal/Subscriber.js:49:17)
super._subscribe (webpack:///node_modules/rxjs/_esm2015/internal/BehaviorSubject.js:14:23)
[2020-11-30 08:30:17.979] [error] Frontend Error: Error: Uncaught (in promise): Error: UnknownError: Internal error opening backing store for indexedDB.open.
Error: UnknownError: Internal error opening backing store for indexedDB.open.
    at t.<anonymous> (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2331429)
    at Generator.throw (<anonymous>)
    at o (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2329740)
    at l.invoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:6639)
    at Object.onInvoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343827)
    at l.invoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:6579)
    at i.run (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:2051)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:16770
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7257)
    at Object.onInvokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343716)
    at Z (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:15998)
    at Z (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:15527)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:16807
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7257)
    at Object.onInvokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343716)
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7178)
    at i.runTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:2666)
    at m (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:9251)
    at u.invokeTask [as invoke] (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:8336)
    at _ (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:20152) Error: Uncaught (in promise): Error: UnknownError: Internal error opening backing store for indexedDB.open.
Error: UnknownError: Internal error opening backing store for indexedDB.open.
    at t.<anonymous> (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2331429)
    at Generator.throw (<anonymous>)
    at o (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2329740)
    at l.invoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:6639)
    at Object.onInvoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343827)
    at l.invoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:6579)
    at i.run (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:2051)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:16770
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7257)
    at Object.onInvokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343716)
    at Z (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:15998)
    at Z (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:15527)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:16807
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7257)
    at Object.onInvokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343716)
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7178)
    at i.runTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:2666)
    at m (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:9251)
    at u.invokeTask [as invoke] (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:8336)
    at _ (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:20152)
[2020-11-30 08:30:31.568] [error] Frontend Error Stack: terminated (webpack:///src/app/core/persistence/database.service.ts:81:12)
Ur (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2329740)
invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:373:29)
onInvoke (webpack:///node_modules/@angular/core/fesm2015/core.js:27488:32)
invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:372:55)
run (webpack:///node_modules/zone.js/fesm2015/zone.js:135:46)
zone.scheduleMicroTask(source, (webpack:///node_modules/zone.js/fesm2015/zone.js:1274:39)
invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:407:34)
onInvokeTask (webpack:///node_modules/@angular/core/fesm2015/core.js:27476:32)
scheduleResolveOrReject (webpack:///node_modules/zone.js/fesm2015/zone.js:1215:38)
value.hasOwnProperty (webpack:///node_modules/zone.js/fesm2015/zone.js:1165:20)
zone.scheduleMicroTask(source, (webpack:///node_modules/zone.js/fesm2015/zone.js:1277:20)
invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:407:34)
onInvokeTask (webpack:///node_modules/@angular/core/fesm2015/core.js:27476:32)
invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:406:63)
task._transitionTo (webpack:///node_modules/zone.js/fesm2015/zone.js:179:50)
drainMicroTaskQueue (webpack:///node_modules/zone.js/fesm2015/zone.js:583:38)
task.zone.runTask (webpack:///node_modules/zone.js/fesm2015/zone.js:492:24)
callback (webpack:///node_modules/zone.js/fesm2015/zone.js:1598:17)
[2020-11-30 08:32:37.296] [error] Frontend Error: Error: Uncaught (in promise): Error: UnknownError: Internal error opening backing store for indexedDB.open.
Error: UnknownError: Internal error opening backing store for indexedDB.open.
    at t.<anonymous> (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2331429)
    at Generator.throw (<anonymous>)
    at o (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2329740)
    at l.invoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:6639)
    at Object.onInvoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343827)
    at l.invoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:6579)
    at i.run (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:2051)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:16770
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7257)
    at Object.onInvokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343716)
    at Z (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:15998)
    at Z (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:15527)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:16807
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7257)
    at Object.onInvokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343716)
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7178)
    at i.runTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:2666)
    at m (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:9251)
    at u.invokeTask [as invoke] (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:8336)
    at _ (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:20152) Error: Uncaught (in promise): Error: UnknownError: Internal error opening backing store for indexedDB.open.
Error: UnknownError: Internal error opening backing store for indexedDB.open.
    at t.<anonymous> (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2331429)
    at Generator.throw (<anonymous>)
    at o (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2329740)
    at l.invoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:6639)
    at Object.onInvoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343827)
    at l.invoke (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:6579)
    at i.run (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:2051)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:16770
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7257)
    at Object.onInvokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343716)
    at Z (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:15998)
    at Z (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:15527)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:16807
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7257)
    at Object.onInvokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:343716)
    at l.invokeTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:7178)
    at i.runTask (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:2666)
    at m (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:9251)
    at u.invokeTask [as invoke] (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:8336)
    at _ (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/polyfills.250ebd3af63450554a58.js:1:20152)
[2020-11-30 08:32:41.182] [error] Frontend Error Stack: terminated (webpack:///src/app/core/persistence/database.service.ts:81:12)
Ur (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2329740)
invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:373:29)
onInvoke (webpack:///node_modules/@angular/core/fesm2015/core.js:27488:32)
invoke (webpack:///node_modules/zone.js/fesm2015/zone.js:372:55)
run (webpack:///node_modules/zone.js/fesm2015/zone.js:135:46)
zone.scheduleMicroTask(source, (webpack:///node_modules/zone.js/fesm2015/zone.js:1274:39)
invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:407:34)
onInvokeTask (webpack:///node_modules/@angular/core/fesm2015/core.js:27476:32)
scheduleResolveOrReject (webpack:///node_modules/zone.js/fesm2015/zone.js:1215:38)
value.hasOwnProperty (webpack:///node_modules/zone.js/fesm2015/zone.js:1165:20)
zone.scheduleMicroTask(source, (webpack:///node_modules/zone.js/fesm2015/zone.js:1277:20)
invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:407:34)
onInvokeTask (webpack:///node_modules/@angular/core/fesm2015/core.js:27476:32)
invokeTask (webpack:///node_modules/zone.js/fesm2015/zone.js:406:63)
task._transitionTo (webpack:///node_modules/zone.js/fesm2015/zone.js:179:50)
drainMicroTaskQueue (webpack:///node_modules/zone.js/fesm2015/zone.js:583:38)
task.zone.runTask (webpack:///node_modules/zone.js/fesm2015/zone.js:492:24)
callback (webpack:///node_modules/zone.js/fesm2015/zone.js:1598:17)
[2020-11-30 08:33:52.932] [error] Frontend Error: Error: No tag TODAY
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2028559
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144622
    at memoized (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144116)
    at lt (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2145023)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144676
    at memoized (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144116)
    at a.project (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2143846)
    at a._next (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:1937661)
    at a.next (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:220941)
    at t._subscribe (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:111996) Error: No tag TODAY
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2028559
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144622
    at memoized (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144116)
    at lt (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2145023)
    at file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144676
    at memoized (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2144116)
    at a.project (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:2143846)
    at a._next (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:1937661)
    at a.next (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:220941)
    at t._subscribe (file:///C:/Users/myusername/AppData/Local/Programs/superProductivity/resources/app.asar/dist/main.be03b267c7a5644f24c6.js:1:111996)
[2020-11-30 08:33:58.200] [error] Frontend Error Stack: webpack:///src/app/features/tag/store/tag.reducer.ts:46:12
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1096:29)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
args (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1059:38)
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1105:35)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
source$.pipe(pluck (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:868:24)
_next (webpack:///node_modules/rxjs/_esm2015/internal/operators/map.js:29:34)
next (webpack:///node_modules/rxjs/_esm2015/internal/Subscriber.js:49:17)
super._subscribe (webpack:///node_modules/rxjs/_esm2015/internal/BehaviorSubject.js:14:23)
```
### Console Output

Running from Start menu, so no console.

### Stacktrace
```
webpack:///src/app/features/tag/store/tag.reducer.ts:46:12
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1096:29)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
args (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1059:38)
function (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1105:35)
memoized (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:1011:38)
source$.pipe(pluck (webpack:///node_modules/@ngrx/store/fesm2015/ngrx-store.js:868:24)
_next (webpack:///node_modules/rxjs/_esm2015/internal/operators/map.js:29:34)
next (webpack:///node_modules/rxjs/_esm2015/internal/Subscriber.js:49:17)
super._subscribe (webpack:///node_modules/rxjs/_esm2015/internal/BehaviorSubject.js:14:23)
```

### Meta Info
META: SP6.1.0-2 Electron – de – Win32 – Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) superProductivity/6.1.0-2 Chrome/85.0.4183.121 Electron/10.1.5 Safari/537.36

### Actions Before Error
```
1606489714943: [Persistence] Save to DB 
1606489714943: [Persistence] Save to DB 
1606489715936: [Task] Add time spent 
1606489715939: [Tag] Update Work End for Tag 
1606489715980: [Persistence] Save to DB 
1606489715983: [Persistence] Save to DB 
1606489716941: [Task] Add time spent 
1606489716953: [Tag] Update Work End for Tag 
1606489716958: [Persistence] Save to DB 
1606489716959: [Persistence] Save to DB 
1606489717155: [Task] Update Task 
1606489717369: [Persistence] Save to DB 
1606489717370: [Task] UnsetCurrentTask 
1606489718358: [Task] SetSelectedTask 
1606489725498: [Task] Move to archive 
1606489725515: [Persistence] Save to DB 
1606489725516: [Persistence] Save to DB 
1606489725516: [Persistence] Save to DB 
1606489725516: [Task] UnsetCurrentTask 
1606489725775: [Persistence] Save to DB 
1606721381447: [SP_ALL] Load(import) all data 
1606721381453: [SP_ALL] All Data was loaded 
1606721381486: [WorkContext] Set Active Work Context 
1606721381714: [Task] SetSelectedTask 
1606721383607: [Task][Issue] Add Task 
1606721383696: [Persistence] Save to DB 
1606721383699: [Persistence] Save to DB 
1606721383708: [TaskRepeatCfg] Update TaskRepeatCfg 
1606721383730: [Persistence] Save to DB 
1606721386029: [Layout] Hide SideBar
```

**Comment by johannesjo** [comment]
> * (Is it perhaps a Monday bug?) 

<3

Thanks for reporting! This should already be fixed (#615) and it's basically a rare race condition. The fix is only available on the edge channel currently. I'll try to provide a new release later this week.

**Comment by sdruskat** [comment]
Perfect, thanks!
